projects/pastanaga/src/lib/styles/_reboot.scss

```diff
@@ -56,12 +56,6 @@ article, aside, dialog, figcaption, figure, footer, header, hgroup, main, nav, s
 // 2. As a best practice, apply a default `background-color`.
 // 3. Set an explicit initial text-align value so that we can later use the
 //    the `inherit` value on things like `<th>` elements.
-html {
```

Comment:
did you discuss it with @albertcasado ?

Reply:
@ebrehault yes, it was his decision. Do I need an approval to merge the PR? Or is not needed in this repo?

Reply:
@LuisPalomo yes you need an explicit approval on pastanaga, it's a bit weird that's not onforced by travis